app/Support/ContentCache.php

```
@@ -46,11 +49,53 @@ public static function deferBooksChanged(): void
+        QueryCache::bumpGeneration('home_');
+    }
+
+    /**
+     * A review was approved, rejected or deleted: the cached public reviews
+     * block ('book_reviews_' namespace, book-detail page) is stale. New
+     * reviews are created as 'pendente' (never publicly visible), so only
+     * moderation transitions need to invalidate.
+     */
+    public static function reviewsChanged(): void
+    {
+        QueryCache::bumpGeneration('book_reviews_');
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'App\Support\QueryCache' in method 'reviewsChanged'. (undefined)

(StaticAccess)